internal/utils/smtp.go

```diff
@@ -0,0 +1,34 @@
+package utils
```

Comment:
Please move this file to the notifier package as suggested.

Reply:
Done.